README.md

```diff
@@ -36,7 +36,6 @@ The above sample waits on a trigger from the queue named "queue" connected to th
 |--|--|--|
 |ConnectionStringSetting|The connection string for the RabbitMQ queue|`amqp://user:password@url:port`|
 |QueueName|The name of the source or destination queue|`myQueue`|

```

Comment:
nit:  Add `To move failed messages to deadletterqueue, configure queue (add link)`